tox.ini

```diff
@@ -35,7 +35,7 @@ setenv =
 
 deps =
     napari_repo: git+https://github.com/napari/napari.git
-extras = testing
```

Comment:
As there is no skip for uv tests I prefer to add uv to testing extras. 

Using `napari-plugin-manager[uv]` in testing extras.